database/raft/raft.go

```diff
@@ -147,6 +147,11 @@ type Getter interface {
 func Start(laddr, dir, bootURL string, httpClient *http.Client) (*Service, error) {
```

Comment:
I know this was a previous PR, but while we're poking at this, can you add `httpClient` to the doc comment for `Start`?